docs/autoupdate.md

```diff
@@ -0,0 +1,44 @@
+# Autoupdate
+
+*Autoupdate* is a feature that will guarantee you are always up to
+date. It is enabled by default, and can be disabled via `snappy config`.
+
+## Usage
+
+To check whether the feature is active, run
+
+    snappy config ubuntu-core | grep autoupdate
+
+If you want to disable it run
+
+    echo 'config: {ubuntu-core: {autoupdate: off}}' | sudo snappy config ubuntu-core
```

Comment:
Eventually we'll need something more friendly for this. Something along the lines of:

    sudo snappy-config ubuntu-core config.ubuntu-core.autoupdate=off